css/views_table_contextual_links_fix.css

```diff
@@ -2,3 +2,14 @@
   width: 34px;
   position: relative;
 }
+
+.islandora-batch-view-table td.views-field-actions .ctools-dropbutton-wrapper {
+  position: relative;
+  width: 100%;
+  display: block;
+}
```

Comment:
Is this section really necessary? Seems to work the same with and without it.

Reply:
Paranoia really, absolute positioning is bound to the first parent whose positioning is not default. This is just ensuring that no matter what happens, the child object will always remain positioned relative to the wrapper span. 
More paranoia, Divs inside spans don't make logical or semantic sense.